data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -38,34 +38,31 @@ def get_survey_question_details(access_token: str, survey_id: str) -> list:
 
 
 def get_no_answer_options_for_question_json() -> dict:
+    """Added for consistency of the schema in BQ"""
```

Comment:
My preference:
I would really prefer if we didn't have a "no answer" option unless it is needed for the end result.
e.g. if we have a subject area filter and need an option saying "No Subject Area".

In this case it doesn't seem to match that.
And it if you counted the "options" it would say `1` even though it should be `0`.

Just adding it because it gets past schema validation errors does not seem to be a good reason to add it.
It would be better to look at the schema validation errors.
Why can't for example `choices` be an empty list or optional?
What is the schema that is being generated?

Reply:
OK